ReactiveObjC/extobjc/EXTRuntimeExtensions.m

```diff
@@ -62,7 +62,7 @@
 
         if (!next) {
             fprintf(stderr, "ERROR: Could not read class name in attribute string \"%s\" for property %s\n", attrString, property_getName(property));
-            return NULL;
+			goto errorOut;
```

Comment:
Could you please fix the indentation?

Reply:
I'm not sure but it seems like a github diff issue. If I sync the project the indentation looks fine and no matter what I do I don't see diff compared to the origin. 🤔

Reply:
ok, it seems I managed to fix it.